pkg/lib/authn/identity/ldap/store.go

```diff
@@ -125,6 +126,51 @@ func (s *Store) List(userID string) ([]*identity.LDAP, error) {
 	return is, nil
 }
 
+func (s *Store) ListByClaim(name string, value string) ([]*identity.LDAP, error) {
+	q := s.selectQuery().
+		Where("(l.claims ->> ?) = ?", name, value)
+
+	rows, err := s.SQLExecutor.QueryWith(q)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+
+	var is []*identity.LDAP
+	for rows.Next() {
+		i, err := s.scan(rows)
+		if err != nil {
+			return nil, err
+		}
+		is = append(is, i)
+	}
+
+	return is, nil
+}
+
+func (s *Store) ListByClaimJSONPointer(pointer jsonpointer.T, value string) ([]*identity.LDAP, error) {
```

Comment:
Please remove this method.
You can see https://github.com/authgear/authgear-server/pull/4531/files

Now we always use ListByClaim

Reply:
I know. I am waiting louis help me rebasing the branch `feature-ldap`.